packages/react/src/Banner/Banner.test.tsx

```diff
@@ -148,8 +148,8 @@ describe('Banner', () => {
       />,
     )
 
-    expect(screen.getByRole('button', {name: 'test primary action'})).toBeInTheDocument()
-    expect(screen.getByRole('button', {name: 'test secondary action'})).toBeInTheDocument()
+    expect(screen.queryAllByRole('button', {name: 'test primary action', hidden: true}).length).toBe(2)
+    expect(screen.queryAllByRole('button', {name: 'test secondary action', hidden: true}).length).toBe(2)
```

Comment:
can you walk me through this? we have to buttons now and they're both hidden?

Reply:
There was always 2 buttons, but I think jest doesn't have access to the CSS modules styles like it did before, so it thinks they're both visible. I only really added the `hidden` for backwards compatibility but I think it's unnecessary here.